server.go

```diff
@@ -187,6 +187,20 @@ func (m *RdmaDevicePlugin) Allocate(ctx context.Context, r *pluginapi.AllocateRe
 		devicesList = append(devicesList, ds)
 	}
 
+	// for /dev/infiniband/rdma_cm
+	rdma_cm_paths := []string{
+		"/dev/infiniband/rdma_cm",
+		"/dev/infiniband/umad0",
```

Comment:
I'm not very sure abount `umad0` and `ucm0` is right here? Maybe we should return `umadX` and `ucmX` devices that are related  with the `uverbX` device. But now, I think only  `rdma_cm` returned  is enough. WDYT?

Reply:
yeah, I am also not sure about those two devices, but it will be needed while doing some tests as [here](https://community.mellanox.com/docs/DOC-2086).  However, sure, we may do it later.